_posts/2019-08-23-gsoc_LibrecoresCI_Report.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+title: "GSoC Project Report: Continuous Integration for Hardware Projects on LibreCores CI"
+description: ""
+category:
+tags: ['GSoC', 'LibrecoresCI' ,'FOSSi-Foundation']
+author: Nancy Chauhan
+---
+
+Librecores provides a platform to share projects and ideas, in the area of free and open-source digital hardware design. Librecores CI is a service to provide CI for hardware projects hosted on Librecores to improve user experience and reliability.
+
+In these summers I worked on developing continuous integration solution for hardware Projects such as OpenRISC, a family of free and open-source processor implementations on the RISC architecture.
+
+I worked from May to August 2019. In this project I worked on contrasting technologies (EDA tools, processors, Docker, Jenkins, Groovy DSL & Jenkins Shared Pipeline library) and GitHub organizations (OpenRISC, LibreCores, FuseSoC ) with the following goals:
+
+1) Recover OpenRISC CI pipelines and modify them to use standard LibreCores images
+
+2) Optimise CI Pipelines with new test automation
```

Comment:
I think a better wording is: **Enhance** CI pipelines... here too. *Optimize* seems to intend you made them faster compared to the original travis pipeline.